src/Stryker.Core/Stryker.Core/Mutators/BlockMutator.cs

```diff
@@ -13,7 +15,53 @@ class BlockMutator : MutatorBase<BlockSyntax>, IMutator
 
         public override IEnumerable<Mutation> ApplyMutations(BlockSyntax node)
         {
-            if (node.Parent is MethodDeclarationSyntax methodDeclaration && methodDeclaration.ReturnType.ToString() != "void")
+            // Note: using local functions below so that the check order can be easily reordered,
+            // if a need to optimize arises at some point.
+
+            bool IsInStructConstructor()
```

Comment:
The ApplyMutations() method is getting a bit big now. Maybe move this local function to a class method?